.markdownlint.yml

```diff
@@ -0,0 +1,7 @@
+default: true
```

Comment:
I personally use VSCode so I am fine with this change but wondering what others think about adding IDE/editor/extension specific config files in the root dir.

Reply:
What do you think about adding `exportloopref` ?

Reply:
see cbe1692504021f71c69af1b636aa00cbe6015285

FYI - it didn't actually catch the problem. But we're also using it in `go-algorand` so it should be safe.